print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
+
+  getNext() {
+    return this.next;
+  }
+
+  setNext(next) {
+    this.next = next;
+  }
+
+  getValue() {
+    return this.value;
+  }
+
+  setValue(value) {
+    this.value = value;
+  }
+
+  printList() {
+    let current = this;
+
+    while (current != null) {
```

Comment:
```suggestion
while (current) {
```

(nit) `null` is falsy in Javascript, which allows the while loop to check through `while(current)`.